sql/v2/README.md

```diff
@@ -0,0 +1,27 @@
+# CloudEvents Expression Language Go implementation
+
+CloudEvents Expression Language implementation.
+
+Note: this package is a work in progress, APIs might break in future releases.
+
+## User guide
+
+To start using it:
+
+```go
+import cesqlparser "github.com/cloudevents/sdk-go/sql/v2/parser"
+
+// Parse the expression
+expression, err := cesqlparser.Parse("subject = 'Hello world'")
```

Comment:
mind adding a bit more of use cases, also more complex ones?

Sure: can be done in a follow up PR